Tasks/HelmDeployV0/task.json

```diff
@@ -393,6 +433,32 @@
             "defaultValue": "true",
             "helpMarkDown": "If this is true, this task will fail if any errors are written to the error pipeline, or if any data is written to the Standard Error stream. Otherwise the task will rely on the exit code to determine failure.",
             "groupName": "advanced"
+        },
+        {
+            "name": "chartNameForACR",
+            "aliases": [
+                "chartName"
+            ],
+            "label": "Chart Name For Azure Container Registry",
+            "type": "string",
+            "helpMarkDown": "Chart name with which the chart will be stored in Azure Container Registry.",
+            "defaultValue": "",
+            "visibleRule": "command == save",
+            "required": "true",
+            "groupName": "commands"
+        },
+        {
+            "name": "chartPathForACR",
```

Comment:
You could use the existing input for chart path, like package command is using. A new input makes sense for chartNameForACR, but not for path. What do you think?

Reply:
Discussed offline and we decided to keep this as the help text is not reusable.